wolfcrypt/src/aes.c

```diff
@@ -1725,12 +1725,14 @@ static WARN_UNUSED_RESULT word32 inv_col_mul(
     #endif
 #endif
 
+#define WOLFSSL_AES_TOUCH_LINES
```

Comment:
Think this macro should be gated by WC_NO_CACHE_RESISTANT, for builds where performance is more important.

Reply:
Still working on the code.
We may not go with it if it is too slow.